packages/mcp-docs-server/src/tools/blog.ts

```diff
@@ -3,91 +3,101 @@ import { z } from 'zod';
 
 // Helper function to fetch blog posts as markdown
 async function fetchBlogPosts(): Promise<string> {
-  try {
-    const response = await fetch('https://mastra.ai/blog');
-    if (!response.ok) {
-      throw new Error('Failed to fetch blog posts');
-    }
-    const html = await response.text();
+  const response = await fetch('https://mastra.ai/blog');
+  if (!response.ok) {
+    throw new Error('Failed to fetch blog posts');
+  }
+  const html = await response.text();
 
-    const dom = new JSDOM(html);
-    const document = dom.window.document;
+  const dom = new JSDOM(html);
```

Comment:
We should look in the future to use happy-dom as it's more lightweight. but nothing to change right now

Reply:
good call! wouldn't take much time, all this JSDOM code was vibe coded in a couple shots and there are good tests for it

Reply:
other alternative is we could expose a static JSON API in the blog and fetch that directly without any server dom